plasmapy/examples/plot_langmuir_analysis.py

```diff
@@ -22,8 +21,8 @@
 # a Electron Energy Distribution Function (EEDF) as well.
 
 # Load the bias and current values stored in the .p pickle file.
-path = os.path.join("langmuir_samples", "Beckers2017.p")
-bias, current = pickle.load(open(path, 'rb'))
+path = os.path.join("langmuir_samples", "Beckers2017.npy")
+bias, current = np.load(open(path, 'rb'))
```

Comment:
Numpy just takes the path here. This way you wouldn't close the file. Actually, you didn't close it in the first implementation anyway, come to think about it :D